src/Altinn.Platform/Altinn.Platform.Register/Altinn.Platform.Register.IntegrationTest/Mocks/Authentication/SigningKeysRetrieverStub.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+using Microsoft.IdentityModel.Tokens;
+
+namespace Altinn.Platform.Storage.IntegrationTest.Mocks.Authentication
+{
+    /// <inheritdoc />
+    public class SigningKeysRetrieverStub : ISigningKeysRetriever
+    {
+        /// <inheritdoc />
+        public async Task<ICollection<SecurityKey>> GetSigningKeys(string url)
```

Comment:
Copy this method into ConfigurationManagerStub, make it private and use it directly. 

Delete this file and the file for the interface. Unnecessary complexity and possible confusion with the intended use in the Authentication component. Mocking of Maskinporten token validation and not JwtCookie validation. 

The SigningKeysRetrieverStub was reused because the tests for both parts use the same certificate. Might have been a mistake now that I see how it spreads around.